powertools-serialization/src/main/java/software/amazon/lambda/powertools/utilities/JsonConfig.java

```diff
@@ -38,11 +36,7 @@ public static JsonConfig get() {
         return ConfigHolder.instance;
     }
 
-    private static final ThreadLocal<ObjectMapper> om = ThreadLocal.withInitial(() -> {
-        ObjectMapper objectMapper = new ObjectMapper();
-        objectMapper.configure(FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
Do we know why this was configured like this?

Reply:
It was configured like this for the tests in validation module, before using the java-lambda-serialization library. See discussion with Pankaj [here](https://github.com/awslabs/aws-lambda-powertools-java/pull/757#discussion_r809905638).